cloudbuild.yaml

```diff
@@ -1,7 +1,7 @@
 substitutions:
   # using this base image for now, because there is an issue causing the newest COS dev
   # image not booting with cs.
-  '_BASE_IMAGE': 'cos-dev-105-17234-0-0'
+  '_BASE_IMAGE': '' # will use the latest cos dev image
```

Comment:
cos-dev is now at the 109 milestone. Do we have changes from cos-dev that we will want to incorporate eventually? If not, then it may be useful to switch to cos-beta for more stability between releases.

Reply:
I am also considering this, on the cos-dev track we may have a (slightly) newer kernel and other dependencies.

Reply:
Switched to cos-beta 105